components/dashboard/src/components/ConfirmationModal.tsx

```diff
@@ -20,7 +20,7 @@ export default function ConfirmationModal(props: {
     onConfirm: () => void;
 }) {
     const children: React.ReactChild[] = [
-        <p key="areYouSure" className="mt-3 mb-3 text-base text-gray-500">
+        <p key="areYouSure" className="mb-3 text-base text-gray-500">
```

Comment:
**praise**: Thanks for making this spacing change! Cross-posting relevant comment from https://github.com/gitpod-io/gitpod/pull/15316#discussion_r1046881412 for more context.